lib/Optimizer/Transforms/DecompositionPatterns.cpp

```diff
@@ -460,6 +460,25 @@ struct R1ToRz : public OpRewritePattern<quake::R1Op> {
   }
 };
 
+// Naive mapping of R1 to U3
+// This is only expected to work with full inlining and
+// quake apply specialization.
```

Comment:
I think this is universal wrt decompositions. Comment isn't needed.

Reply:
I am relying on Preadnya's PR here

Reply:
I have now fixed it in PR#2401 with 09011e866360f1bedfe0070c762ab3cc393b2503